### ISSUE TITLE
[Feature Request] Add a clipboard manager extension

### ISSUE THREAD

**Issue by nevara** [oliverschwendener/ueli#85]
add a simple clipboard history feature a la Alfred

**Comment by LalaGhost** [comment]
If you do not have any problem with adding one more program to your PC, you can download Ditto, works perfect for me.
Press [your registered keystroke], this opens this small window:
![screenshot_2](https://user-images.githubusercontent.com/43962138/50782492-74821a80-12a8-11e9-9d0d-0d1e46bbe537.png)
Then you have several different way to paste something.
1. Press CTRL + a number from 1 to 9 to paste its corresponding entry within Ditto move it into your clipbaord
2. Using arrow keys going up and down to chose the entry you want to paste and move it into your clipbaord
3. Search within Ditto after one past copied text and paste it via 1. or 2.

https://ditto-cp.sourceforge.io/

**Comment by nevara** [comment]
That's true. Unfortunately I work for a corporation, so we are not using the latest Windows 10 version. And Cloud Clipboard would be deactivated anyway. 

I don't like Ditto's interface and would rather have everything integrated.

**Comment by Ayush-Pandey-Programmer** [comment]
Windows 10 has a already clipboard manager hit Win+V to launch that it is better than downloading a another app to our pc if ueli have clipboard feature this is also a "cherry on the cake"

**Comment by oliverschwendener** [comment]
I'm closing this feature request as I have no plans to implement this. If anyone else wants to do it, feel free to reopen.

**Comment by asdwsxzc123** [comment]
I fork this project,add a clipboard manager extension. 

![Image](https://github.com/user-attachments/assets/f46231ef-3189-47bf-9b7e-d7768d533690)
